lib/internal/test_runner/utils.js

```diff
@@ -368,13 +369,46 @@ function getCoverageReport(pad, summary, symbol, color, table) {
     }
   }
 
+  function getfilePathMultiline(string, width, pad, coverage) {
+    if (!table) return string;
+  
+    const lines = [];
+    let currentLine = '';
+    
+    for (const word of string.split('\\')) {
+      if (currentLine.length + word.length + 1 <= width) {
+        // If adding the next word fits in the current line, append it
+        currentLine += (currentLine.length > 0 ? '\\' : '') + word;
+      } else {
+        // If adding the next word exceeds the width, start a new line
+        lines.push(currentLine);
+        currentLine = word;
+      }
+    }
+  
+    // Add the last line if any
+    if (currentLine.length > 0) {
+      lines.push(currentLine);
+    }
+  
+    const truncatedLines = lines.map(line => line.slice(0, width));
+  
+    let result = truncatedLines.join('\n');
```

Comment:
```suggestion
    let result = ArrayPrototypeJoin(truncatedLines, '\n');
```

Reply:
I also thought about using primordials earlier the only thing stopping me was will it behave the same as for string as it is behaving for arrays but thanks for clarifying, I'll make the changes ASAP. Appreciate the feedback :)